src/screens/SearchPlaces.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import { GooglePlacesAutocomplete } from "react-native-google-places-autocomplete";
+import MapView from "react-native-maps";
+import {
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+} from "react-native";
+import { COLOURS } from "../constants";
+import ProceedIcon from "../../assets/SVG/proceedIcon";
+
+
+const SearchPlacesScreen = () => {
+  const [location, setLocation] = useState({
+    latitude: 23.83215,
```

Comment:
from where these constant values are coming?

Reply:
This is a random data , will change it later on